src/api/generate-csv/map-application-to-csv/helpers/csv-new-line/index.ts

```diff
@@ -0,0 +1,3 @@
+const NEW_LINE = '\r\n';
```

Comment:
Should this be under constant instead of a helper?

Reply:
We could do, does make sense, but I think we should keep in the "map application to CSV" directory because it's specific for the CSV and this CSV generation is a temporary/MVP approach that will be removed when we integrate with TFM. So when it comes to that we could delete the CSV generation functionality in a single directory. What do you think?